src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -452,6 +452,8 @@ public interface JavaLangAccess {
 
     Object stringConcat1(String[] constants);
 
+    byte stringInitCoder();
```

Comment:
Can you add javadoc comment about on this API, like the javadoc for other JLA methods?